test/blocks/video/video.test.js

```diff
@@ -4,20 +4,45 @@ import sinon from 'sinon';
 import { waitFor, waitForElement } from '../../helpers/waitfor.js';
 
 import { setConfig, createTag } from '../../../libs/utils/utils.js';
-import { decorateAnchorVideo } from '../../../libs/utils/decorate.js';
+import { decorateAnchorVideo, handlePause, applyHoverPlay, decoratePausePlayWrapper } from '../../../libs/utils/decorate.js';
 
 setConfig({});
 const { default: init } = await import('../../../libs/blocks/video/video.js');
 
 describe('video uploaded using franklin bot', () => {
+  let clock;
   beforeEach(async () => {
+    clock = sinon.useFakeTimers({
+      toFake: ['setTimeout'],
+      shouldAdvanceTime: true,
+    });
     document.body.innerHTML = await readFile({ path: './mocks/body.html' });
   });
 
   afterEach(() => {
+    clock.restore();
     document.body.innerHTML = '';
   });
 
+  it('aria-label should not have index when page has only one video', async () => {
+    const block = document.querySelector('.video.autoplay.single');
+    const block2 = document.querySelector('.video.autoplay.second');
+    const a = block.querySelector('a');
+    const a2 = block2.querySelector('a');
+    block.append(a);
+    block2.append(a2);
```

Comment:
looks like `document.querySelector('.video.autoplay.single')` is a single div with an `a` tag inside, why are you appending the item to the same block it is found in? 

Also, arguably, we should stick to semantic naming conventions even in our tests.